testdata/expected/Gemfile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016 Google Inc.
```

Comment:
Today is 2018, actually.
Here and in the other files.

Reply:
If we're going to change the date, we should probably also change the copyright holder since my understanding that it should now be Google LLC.